src/Microsoft.DotNet.Configurer/DotnetFirstTimeUseConfigurer.cs

```diff
@@ -36,9 +36,22 @@ public void Configure()
         private void PrintFirstTimeUseNotice()
         {
             const string firstTimeUseWelcomeMessage = LocalizableStrings.FirstTimeWelcomeMessage;
+            const string firstTimeTelemetryEnabledMessage = LocalizableStrings.FirstTimeTelemetryEnabledMessage;
+            const string firstTimeConfiguringMessage = LocalizableStrings.FirstTimeConfiguringMessage;
 
             Reporter.Output.WriteLine();
             Reporter.Output.WriteLine(firstTimeUseWelcomeMessage);
+
+            var telemetryEnabled = 
+                _environmentProvider.GetEnvironmentVariableAsBool("DOTNET_CLI_TELEMETRY_OPTOUT", false);
+            
+            if (telemetryEnabled) {
+                Reporter.Output.WriteLine();
+                Reporter.Output.WriteLine(firstTimeConfiguringMessage);
```

Comment:
This should be `Reporter.Output.WriteLine(firstTimeTelemetryEnabledMessage);`

Oops.